### DISCUSSION TITLE
Azure student partner offer not working

### DISCUSSION THREAD

**Discussion by saidibyanshuiitbcse** [community/community#147184] [GitHub Education]
### Hi everyone,

![image](https://github.com/user-attachments/assets/8a9a8787-8e8d-4b9f-9d6f-0acb1bfd8fad)
Although i am verified in github student dev plan.

**Comment by davevad93** [comment]
Hi @saidibyanshuiitbcse ,

This is a known issue that has been solved here #135739 , you have the following options:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

I recommend the first one.
